### PR TITLE
Add support for externalCerts in console

### PR DESCRIPTION
**Pull request by nitisht**

### CODE REVIEW

**Review by harshavardhana — DISMISSED** [review]

File: helm/minio-operator/templates/tenant.yaml [hunk 1]
```diff
@@ -122,4 +122,12 @@ spec:
     env:
       {{ toYaml . | nindent 6 }}
     {{- end }}
+    {{- with .externalCaCertSecret }}
+    externalCaCertSecret:
+      {{ toYaml . | nindent 6 }}
+    {{ end }}
+    {{- with .externalCertSecret }}
+    externalCaCertSecret:
```
Comment (vadmeste) [inline helm/minio-operator/templates/tenant.yaml:130]: Shouldn't this be externalCertSecret ?

Comment (harshavardhana) [reply helm/minio-operator/templates/tenant.yaml:130]: Good catch..

Comment (nitisht) [reply helm/minio-operator/templates/tenant.yaml]: fixed

**Review by harshavardhana — DISMISSED** [review]

**Review by harshavardhana — APPROVED** [review]

**Review by vadmeste — APPROVED** [review]
LGTM

**Comment by jawabuu** [comment]
Hey @nitisht @vadmeste 
How can I make use of this updated chart?
I was about to file an issue on console ignoring env values but found this.

**Comment by nitisht** [comment]
You can clone the current master or wait for a release

**Pull request merged**